library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -2220,6 +2220,41 @@ void libspdm_reset_context(void *context)
     #endif /* LIBSPDM_ENABLE_MSG_LOG */
 }
 
+/**
+ * Free the memory of contexts within the SPDM context.
+ * These are typically contexts whose memory has been allocated by the cryptography library.
+ * This function does not free the SPDM context itself.
+ *
+ * @param[in]  spdm_context         A pointer to the SPDM context.
+ * @param[in]  is_requester         Is the function called from a requester.
+ *
+ */
+void libspdm_deinit_context(void *context, bool is_requester)
```

Comment:
Can we add `is_requester` to the spdm_context?

The requester set it in get_version is called.

Then we dont need to bother the integrate to input the value.

Reply:
See #390.

Reply:
I do not suggest change all functions.

I just request to not add more in `libspdm_deinit_context()`.

Reply:
Let's discuss this at the Monday meeting.

Reply:
@jyao1 explained it at the meeting. Need to make sure that the internal `is_requester` bit gets set properly and robustly.

Reply:
Agree. I think we need set is_requester in VCA cache in spdm_emu.